clang/test/CIR/Lowering/ThroughMLIR/switch.cir

```diff
@@ -0,0 +1,141 @@
+// RUN: cir-opt %s -cir-to-mlir | FileCheck %s -check-prefix=MLIR
+// RUN: cir-opt %s -cir-to-mlir -cir-mlir-to-llvm | mlir-translate -mlir-to-llvmir | FileCheck %s -check-prefix=LLVM
```

Comment:
Thanks for writing this testcase. I'd prefer if you can write this with source code instead (see `clang/test/CIR/Lowering/ThroughMLIR/vectype.cpp`), because this will make this test easier to update when CIR changes. You don't really need to check the LLVM output, but if you want to, try adding another RUN line that uses `-emit-llvm` for that.